frontend/src/components/Details/ActionBar.vue

```diff
@@ -15,13 +15,37 @@ const emitter = inject<Emitter<Events>>("emitter");
 const auth = storeAuth();
 const emulation = ref(false);
 const playInfoIcon = ref("mdi-play");
-const emulationSupported = props.rom.platform_slug.toLowerCase() in platformSlugEJSCoreMap;
+const emulationSupported =
+  props.rom.platform_slug.toLowerCase() in platformSlugEJSCoreMap;
 
 function toggleEmulation() {
   emulation.value = !emulation.value;
   playInfoIcon.value = emulation.value ? "mdi-information" : "mdi-play";
   emitter?.emit("showEmulation", null);
 }
+
+async function copyDownloadLink(rom: Rom) {
+  const downloadLink = await romApi.downloadRom({
+    rom,
+    files: downloadStore.filesToDownloadMultiFileRom,
+    shareLink: true,
+  });
```

Comment:
Yeah this should be a call to a util file or something, it doesn't need to pollute `downloadRom` function

Reply:
Yeah you are right, I'll move it to a generate link function in utils